src/omegga/player.ts

```diff
@@ -361,32 +361,34 @@ class Player implements OmeggaPlayer {
       { first: 'index', timeoutDelay: 500 }
     );
 
+    if(pawn === "None") return;
```

Comment:
The omegga codebase uses [Prettier](https://prettier.io/) for code formatting; you may want to install the extension in your editor and run the formatter, then commit. I noticed because there is no space between `if` and `(` (sorry to nitpick! I just know this will probably get caught by some other stray commit in the future).